docs/en/advanced_tutorials/manager_mixin.md

```diff
@@ -0,0 +1,77 @@
+# Global manager (ManagerMixin)
+
+During training process, it is inevitable that we need to access some variable globally. Here are some examples:
+
+- Accessing the [logger](mmengine.logging.MMLogger) in model to print some initialization information
+- Accessing the [Visualizer](mmengine.config.Config) anywhere to visualize the predictions and feature maps.
+- Accessing the scope in [Registry](mmengine.registry.Registry) to get the current scope.
+
+In order to unify the mechanism to get the global variable built from different classes, MMEngine designs the [ManagerMixin](mmengine.utils.ManagerMixin).
+
+## Interface introduction
+
+- get_instance(name='', \*\*kwargs): Create or get the instance by name.
+- get_current_instance(): Get the currently built instance.
+- instance_name: Get the name of the instance.
+
+## How to use
+
+1. Define a class inherited from `ManagerMixin`
+
+```python
+from mmengine.utils import ManagerMixin
+
+
+class GlobalClass(ManagerMixin):
+    def __init__(self, name, value):
+        super().__init__(name)
+        self.value = value
+```
+
+```note
+Subclasses of `ManagerMixin` must accept `name` argument in `__init__`. The `name` argument is used to identify the instance, and you can get the instance by `get_instance(name)`.
+```
+
+2. Instantiate the instance anywhere. let's take the hook as an example:
+
+```python
+from mmengine import Hook
+
+class CustomHook(Hook):
+    def before_run(self, runner):
+        GlobalClass.get_instance('mmengine', value=50)
+        GlobalClass.get_instance(runner.experiment_name, value=100)
+```
+
+`GlobalClass.get_instance({name})` will first check whether the instance with the name `{name}` has been built. If not, it will build a new instance with the name `{name}`, otherwise it will return the existing instance. As the above example shows, when we call `GlobalClass.get_instance('mmengine')` at the first time, it will build a new instance with the name `mmengine`. Then we call `GlobalClass.get_instance(runner.experiment_name)`, it will also build a new instance with a different name.
```

Comment:
```suggestion
`GlobalClass.get_instance({name})` will first check whether the instance with the `{name}` has been built. If not, it will build a new instance with the `{name}`, otherwise it will return the existed instance. As the above example shows, when we call `GlobalClass.get_instance('mmengine')` for the first time, it will build a new instance with the name `mmengine`. Then we call `GlobalClass.get_instance(runner.experiment_name)`, it will also build a new instance with a different name.
```